extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -502,13 +492,65 @@ private function getCuratedContentQualityForWiki( $wikiID ) {
 	 *
 	 * @return bool
 	 */
-	static function onCuratedContentSave( $sections ) {
-		self::purgeMethod( 'getList' );
```

Comment:
So.. As you're removing this call - is `self::purgeMethod` is being used anywhere?